gkeep_sync/sync_note.py

```diff
@@ -0,0 +1,163 @@
+import os
+import hashlib
+import json
+import requests
+
+from .utils import benchmark, traverse_files, DEFAULT_CONFIG_PATH
+
+SYNC_LABEL = 'autosync'
+FILE_EXTENSION = 'md'
+
+
+class GkeepSyncAPI:
+    def __init__(self, keep):
+        config = json.load(open(DEFAULT_CONFIG_PATH))
+        config['notes_root'] = os.path.expanduser(config['notes_root'])
+
+        self.config = config
+        self.keep = keep
+
+    def create_note(self, title, text):
+        print(f"Creating a note {title}")
+        newNote = self.keep.createNote(title, text)
+        label = self.keep.findLabel(SYNC_LABEL)
```

Comment:
should you get this label only once(probably in __init__)?

Reply:
fair enough